build-in-container/mariner-docker-run.sh

```diff
@@ -0,0 +1,121 @@
+    -v sysfs:/temp/DockerStage/docker-chroot-6/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-6/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-7/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-7/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-7/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-7/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-7/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-7/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-7/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-8/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-8/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-8/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-8/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-8/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-8/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-8/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-9/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-9/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-9/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-9/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-9/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-9/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-9/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-10/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-10/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-10/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-10/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-10/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-10/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-10/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-11/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-11/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-11/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-11/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-11/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-11/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-11/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/temp/DockerStage/docker-chroot-12/upstream-cached-rpms:rw \
+    -v $(pwd)/ccache:/temp/DockerStage/docker-chroot-12/ccache-dir:rw \
+    -v /dev:/temp/DockerStage/docker-chroot-12/dev:ro \
+    -v /proc:/temp/DockerStage/docker-chroot-12/proc:ro \
+    -v devpts:/temp/DockerStage/docker-chroot-12/dev/pts:ro \
+    -v sysfs:/temp/DockerStage/docker-chroot-12/sys:ro \
+    -v tmpfs:/temp/DockerStage/docker-chroot-12/run:ro \
+    -v $(pwd)/build/rpm_cache/cache:/tmp/mariner/build/rpm_cache/cache:rw \
```

Comment:
What is `$(pwd)` here? It seems to be the tutorial repo (since we need to be here to call these functions) which results in hundreds of random files being added (this makes git very sad).

This should point to the directory we called from, which should be independent of the layout of these files.

Reply:
Made a distinction between repo_dir (Mariner GitHub) and tool_dir (repo containing build-in-container tool). repo_dir defaults to tool_dir if not provided.